build.gradle

```diff
@@ -18,11 +15,9 @@ apply plugin: 'org.flywaydb.flyway'
 
 jar {
     baseName = "Identity-service"
-    version = "0.1.0"
```

Comment:
This won't be versioned like a library JAR would be, and if it were you would usually use the default of `project.version`.